backend/app/models/device.py

```
@@ -0,0 +1,19 @@
+
+class SmartDevice(Base):
+    __tablename__ = "smart_devices"
+
+    user_id: Mapped[int] = mapped_column(ForeignKey("users.id"))
+    name: Mapped[str] = mapped_column(String)
+    entity_id: Mapped[str] = mapped_column(String)
+    device_type: Mapped[Optional[str]] = mapped_column(String, nullable=True)
+    room: Mapped[Optional[str]] = mapped_column(String, nullable=True)
+
+    # Relationships
+    user: Mapped["User"] = relationship(back_populates="devices")
```

Ruff (0.14.5)

Undefined name `User`

(F821)